### PR TITLE
feat: support local storage

### PR DESCRIPTION
**Pull request by sh1luo**
Signed-off-by: sh1luo <690898835@qq.com>

### CODE REVIEW

**Comment by sh1luo** [comment]
Demo link: http://testb.rescld.cn/

**Comment by hsluoyz** [comment]
@Kininaru @WindSpiritSR plz review

File: object/storage.go [hunk 1]
```diff
@@ -17,24 +17,16 @@ package object
 import (
 	"bytes"
 	"fmt"
-
 	awss3 "github.com/aws/aws-sdk-go/service/s3"
 	"github.com/casbin/casdoor/util"
+	"github.com/casbin/casdoor/util/filesystem"
 	"github.com/qor/oss"
 	"github.com/qor/oss/aliyun"
 	//"github.com/qor/oss/qiniu"
```
Comment (WindSpiritSR) [inline object/storage.go:25]: Remove useless packages

**Review by WindSpiritSR — CHANGES_REQUESTED** [review]

File: object/storage.go [hunk 2]
```diff
@@ -17,24 +17,15 @@ package object
 import (
 	"bytes"
 	"fmt"
-
 	awss3 "github.com/aws/aws-sdk-go/service/s3"
 	"github.com/casbin/casdoor/util"
+	"github.com/casbin/casdoor/util/filesystem"
 	"github.com/qor/oss"
 	"github.com/qor/oss/aliyun"
-	//"github.com/qor/oss/qiniu"
```
Comment (Kininaru) [inline object/storage.go:25]: Please set your Goland -> Editor -> Code Style -> Go like this: 
image

**Review by Kininaru — CHANGES_REQUESTED** [review]
The function works well, and maybe we should:

1. Add a explanation in README or in doc site to tell users to add `client_max_body_size 20m;` in Nginx http statement, to upload large pictures.
2. It looks like that path strings have some problems. Maybe we can fix that later.

**Comment by sh1luo** [comment]
@Kininaru ok, fixed.

**Review by Kininaru — APPROVED** [review]

**Review by WindSpiritSR — APPROVED** [review]

**Comment by hsluoyz** [comment]
@sh1luo plz refactor the PR and implement the `oss.StorageInterface` interface for local file system: https://github.com/casbin/casdoor/commit/22d021d17319c98b7fde053aadceedc135accdaf

**Comment by sh1luo** [comment]
@hsluoyz ok, done, demo link: http://testb.rescld.cn/

**Pull request merged**